generators/common/templates/README.md.jhi.ejs

```diff
@@ -138,6 +138,64 @@ Create a `ROLE_ADMIN` and `ROLE_USER` group and add users into them. Modify e2e
 Navigate to **API** > **Authorization Servers**, click the **Authorization Servers** tab and edit the default one. Click the **Claims** tab and **Add Claim**. Name it "groups", and include it in the ID Token. Set the value type to "Groups" and set the filter to be a Regex of `.*`.
 
 After making these changes, you should be good to go! If you have any issues, please post them to [Stack Overflow](https://stackoverflow.com/questions/tagged/jhipster). Make sure to tag your question with "jhipster" and "okta".
+
+### Auth0
+
+If you'd like to use [Auth0](https://auth0.com/) instead of Keycloak, you can follow the below configuration steps:
```

Comment:
I will also prefer to remove the below documentation and instead point to the one documented on https://www.jhipster.tech/security/#oauth2 section